features/step_definitions/basic_step.rb

```diff
@@ -0,0 +1,37 @@
+Then(/^show me the page$/) do
+  save_and_open_page
+end
+
+When(/^I visit "([^"]*)" page$/) do |page|
+  visit root_path
+end
+
+Then(/^I should be on "([^"]*)" page$/) do |page_name|
+  case page_name
```

Comment:
`case` and `when` should be aligned no need for the indentation here.. [Checkout the style guide](https://github.com/bbatsov/ruby-style-guide#indent-when-to-case)